api/src/handlers/epochs.rs

```diff
@@ -0,0 +1,48 @@
+use crate::context::WrappedContext;
+use crate::utils::response_error;
+use log::{error, info};
+use serde::{Deserialize, Serialize};
+use store::epochs::get_epochs;
+use warp::{http::StatusCode, reply::json, Reply};
+use store::dto::EpochInfo;
+
+#[derive(Serialize, Debug, utoipa::ToSchema)]
+pub struct ResponseEpochs {
+    epochs: Vec<EpochInfo>,
+}
+
+#[derive(Deserialize, Serialize, Debug)]
+pub struct QueryParams {
```

Comment:
Are empty `QueryParams` needed? Not sure if it's a requirement from the framework, but it seems unnecessary/confusing.

Reply:
my bad I didn't close this PR after sync with LJ. This endpoint is no longer needed since the epoch's boundaries are set in stakes-etl now. Thank for review tho